### DISCUSSION TITLE
General Guidance : Is it possible to generate FHIR test data for a specific disease module

### DISCUSSION THREAD

**Discussion by neerajshri78** [synthetichealth/synthea#1510] [Q&A]
### Requested Feature

Hi Currently when we run the commands like run_synthea -p 20
It generates data for 20 patients for multiple disease conditions based on the modules.

Is it possible to generate data for only one specific disease for example covid-19

That i will have all patients who had covid

**Comment by jawalonoski** [comment]
One way to do this is to use a [Keep Patients Module](https://github.com/synthetichealth/synthea/wiki/Keep-Patients-Module) that specifies the disease of interest.

Do **not** use the [M Feature](https://github.com/synthetichealth/synthea/wiki/The--M-Feature).
